handbook/engineering/distribution/onboarding.md

```diff
@@ -0,0 +1,81 @@
+# Distribution Engineer Oboarding
+
+Welcome to the distribution team! This document will guide you through distribution-specific onboarding tasks to make your start here at Sourcegraph as smooth as it can be.
```

Comment:
Is there an engineering onboarding guide that we can say they should have completed first?

https://about.sourcegraph.com/handbook/engineering/onboarding

And there's per-role ones...
[Software engineer onboarding](https://about.sourcegraph.com/handbook/engineering/onboarding/software-engineer-onboarding)
[Engineering manager onboarding](https://about.sourcegraph.com/handbook/engineering/onboarding/engineering-manager-onboarding)